src/client.coffee

```diff
@@ -363,19 +358,20 @@ class Client
 
   @api public
   ###
-  registerCheckout: (sessionId, arg1, arg2) ->
+  registerCheckout: (sessionId, args...) ->
```

Comment:
This method doesn't use options at all so it just needs an optional callback parameter.